<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test_form</class>
 <widget class="QWidget" name="test_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="table_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>901</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>901</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="add_button">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="update_button">
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="remove_button">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exit_button">
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>207</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="combo_box_column_name">
      <property name="placeholderText">
       <string>Название колонки для поиска</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_edit_column_data">
      <property name="placeholderText">
       <string>Строка для поиска</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_edit_search">
      <property name="placeholderText">
       <string>Глобальный поиск</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>797</x>
     <y>60</y>
     <width>111</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="push_button_prev_page">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="push_button_next_page">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
